targets/TARGET_NORDIC/TARGET_NRF5/us_ticker.c

```diff
@@ -198,6 +200,35 @@ void common_rtc_set_interrupt(uint32_t us_timestamp, uint32_t cc_channel,
     nrf_rtc_cc_set(COMMON_RTC_INSTANCE, cc_channel, RTC_WRAP(compare_value));
     nrf_rtc_event_enable(COMMON_RTC_INSTANCE, int_mask);
 }
+
+void common_rtc_set_interrupt(uint32_t us_timestamp, uint32_t cc_channel,
+                              uint32_t int_mask)
+{
+    uint32_t prev_overflows;
+
+    while (1)
```

Comment:
Anyway this won't fix overflow issue for setting a timieout. Overflow might occurs during executing of internal_common_rtc_set_interrupt for instance. But preserve time readout is  problem as well, see https://github.com/ARMmbed/mbed-os/issues/4026